s26856-DawidCybuch/zadanie_1.txt

```diff
@@ -0,0 +1,91 @@
+Zadanie 1. Uruchom terminal.
+
+Uruchamiam cmd i loguje się do Szuflandi za pomocą ssh s26856@szuflandia.pjwstk.edu.pl
+Znajduje się właśnie na szuflandi.
+Spodziewałem się takiego wyniku.
+
+Zadanie 2. Wyświetl zawartość aktualnego katalogu.
+
+Wykonuje to za pomocą komendy ls.
+Pokazuje mi zawartość aktualnego katalogu, w moim przypadku PRRG oraz UKOS.
+Tutaj również spodziewałem się takiego wyniku.
+
+Zadnie 3. Sprawdz gdzie aktualnie się znajdujesz.
+
+Wpisuje komende pwd, znajduję się aktualnie w /home/PJWSTK/s26856
+Wynik spodziewany.
+
+Zadanie 4. Przejdzo do katalogu pulpit za pomocą ścieżki względnej.
+
+Znajduję się w katalogu pulpit
+Wynik spodziewany.
+
+
+Zadanie 5. Sprawdź gdzie jesteś. 
+
+Wpisuje komende pwd jestem na /home/PJWSTK/s26856
+
+wynik spodziewany
+
+Zadanie 6. Wyświetl zawartość bieżącego katalogu.
+
+Wykonuje komende ls i widze żę mam w swoim katalogu dwa foldery, PPRG i UKOS.
+Spodziewałem się tego wyniku./
+
+Zadanie 7. Wyświetl zawartość swojego katalogu domowego bez przechodzenia do niego (ls z odpowiednim parametrem).
+
+Wpisuje komendę ls -pa | grep /. Wypisuje mi zawartość moich folderów
+
+Spodziewałem się tego wyniku.
+
+Zadanie 8. Przejdź do katalogu Obrazy (lub innego katalogu, który jest dostępny - podaj wprost,
+ że zmieniasz katalog na inny bo np. Obrazy nie są u Ciebie dostępne) w twoim katalogu domowym za pomocą ścieżki względnej.
+
+
+
+Wchodzę do folderu UKOS komendą cd UKOS.
+Widzę że jestem w folderze ukos.
+Spodziewałem się takiego wyniku.
+
+Zadanie 9. Sprawdź gdzie jesteś.
+
+Komenda pwd, jestem w /home/PJWSTK/s26856/UKOS.
+wykonuje cd żeby wrócić do folderu /home
+Jestem w folderze UKOS.
+Spodziewałem się aktualnego wyniku.
+
+Zadanie 10. Zobacz do jakiego katalogu przejdziesz za pomocą ścieżki ././././././
+
+Użyłem komendy cd ./././././././././. Użycie tej komendy sprawiło że nie przeniosłem się do żadnego folderu ponieważ linux
+ jest skonstruowany że każdy katalog jest odznaczany ./. Czyli użycie tej komendy nie zmienia nic 
+Dodanie dodaktowych ././ - osobiście dodałem ich około 40, spowodowało że nadal jest w tym samym directory.
```

Comment:
Ale system musiał mimo wszystko to przeliczyć. Więc wynikowo stan się nie zmienił lecz ścieżka musiała być przeliczona